### ISSUE TITLE
p-dropdown: Missing ariaLabel when in editable = true

### ISSUE THREAD

**Issue by WTW-Steve-Martin** [primefaces/primeng#15122]
### Describe the bug

The aria Label is not being applied when `[editable]="true"` is set on a dropdown.

The following code correctly applies the aria-label...
```
<p-dropdown
  appendTo="body"
  [options]="pageVariables"
  optionLabel="name"
  optionValue="name"
  placeholder="Select variable"
  ariaLabel="Select variable"
  [(ngModel)]="connector.variableName"
  (onBlur)="onConnectorVariableNameChanged($event)"
></p-dropdown>
```

However, when I add the `[editable]="true"` property the aria-label is not set and Axe Dev Tools reports it missing. e.g.
```
<p-dropdown
  appendTo="body"
  [options]="pageVariables"
  [editable]="true"
  optionLabel="name"
  optionValue="name"
  placeholder="Select variable"
  ariaLabel="Select variable"
  [(ngModel)]="connector.variableName"
  (onBlur)="onConnectorVariableNameChanged($event)"
></p-dropdown>
```

### Environment

Windows 10

### Angular version

17.2.3

### PrimeNG version

17.11.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

### Steps to reproduce the behavior

1. Install the Axe Dev Tools for Accessibility plug-in.
2. Add a `p-dropdown` to your page.
3. Scan the page with Axe
4. The Axe scan passes
5. Set it editable with the `[editable]="true"` property
6. Scan the page with Axe
7. The Axe scan fails reporting the following...

![image](https://github.com/primefaces/primeng/assets/153538315/66456d3a-24c7-4a62-ba6b-8fef9d47a645)

### Expected behavior

Ensure the aria-label is applied when editable.
